eng/scripts/Install-GitHooks.ps1

```diff
@@ -46,10 +46,8 @@ foreach ($file in $hookFiles) {
     Copy-Item -Path $file.FullName -Destination $targetFile -Force
```

Comment:
@hallipr remind me to get more info about the usages of githooks here as I'm generally not a fan of them.